src/System.Drawing.Common/src/System/Drawing/Graphics.cs

```diff
@@ -317,6 +609,10 @@ public void MultiplyTransform(Matrix matrix, MatrixOrder order)
             {
                 throw new ArgumentNullException(nameof(matrix));
             }
+            if (matrix.nativeMatrix == IntPtr.Zero)
+            {
+                return;
+            }
```

Comment:
Why was this added?

Reply:
This fixes the test `MultiplyTransform_DisposedMatrix_Nop` with libgdiplus